notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>You could add:</p><p> </p><p>Basis vectors are tangent vectors that depend on the tangent space $T_pM$ to which they belong, i.e. they depend on the point $p$ of the manifold. Thus, we should actually write $e_i(p)$, but in practice the point $p$ is omitted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>added!</p>